src/app/shared/object-collection/shared/mydspace-item-submitter/item-submitter.component.spec.ts

```diff
@@ -60,10 +60,17 @@ describe('ItemSubmitterComponent', () => {
     }));
   });
 
-  it('should show a badge with submitter name', () => {
+  it('should show a badge with submitter name', fakeAsync(() => {
+
     const badge = fixture.debugElement.query(By.css('.badge'));
 
-    expect(badge).not.toBeNull();
-    expect(badge.nativeElement.innerHTML).toBe(EPersonMock.name);
-  });
+    fixture.whenStable().then(() => {
```

Comment:
why have you changed in this way ? the previous way works as well

moreover can you add a test to check the `N/A` value? thanks